src/services/api.js

```diff
@@ -19,3 +19,11 @@ export async function fetchRestaurants({ regionName, categoryId }) {
   const data = await response.json();
   return data;
 }
+
+export async function fetchRestaurant({ id }) {
+  const url = 'https://eatgo-customer-api.ahastudio.com/restaurants'
+    + `/${id}`;
```

Comment:
전체 url에 template literal을 적용하는건 어떨까요?
```suggestion
  const url = `https://eatgo-customer-api.ahastudio.com/restaurants/${id}`;
```